src/utilities/GlobalConstants.js

```diff
@@ -0,0 +1,10 @@
+export const REGEX_MAPPINGS = Object.freeze({
+  ResourceInStandby: /Base\.(\d+\.){3}ResourceInStandby/,
```

Comment:
I would suggest to keep the keys as Camel Case. eg: resourceInStandby.

Reply:
Sure, will do that.